SCTLD_meta_analysis_metadata.csv

```diff
@@ -1523,4 +1523,4 @@
 22,,,,Salerno,FDEP SCTLD,JSEP_SCTLD_Endo,238H-DLAB-DR,2017,,42517,April,,ENVO:00000150,,,USA,Dustan Rocks,DR,,,,,,,,,"coral skeleton, coral tissue, coral mucus",,,,,,,,,,,,,,,,,,,,y,DNeasy PowerSoil Kit,V4,515F/806R,,,,MiSeq,,Genewiz (NJ),
 23,,,,Salerno,FDEP SCTLD,JSEP_SCTLD_Endo,KIT-BLANK-BACARC,n/a,,n/a,n/a,,ENVO:00000150,,,USA,n/a,n/a,,,,,,,,,"coral skeleton, coral tissue, coral mucus",,,,,,,,,,,,,,,,,,,,y,DNeasy PowerSoil Kit,V4,515F/806R,,,,MiSeq,,Genewiz (NJ),
 24,,,,Salerno,FDEP SCTLD,JSEP_SCTLD_Endo,PCR-NEG-BACARC,n/a,,n/a,n/a,,ENVO:00000150,,,USA,n/a,n/a,,,,,,,,,"coral skeleton, coral tissue, coral mucus",,,,,,,,,,,,,,,,,,,,y,DNeasy PowerSoil Kit,V4,515F/806R,,,,MiSeq,,Genewiz (NJ),
-25,,,,Salerno,FDEP SCTLD,JSEP_SCTLD_Endo,PCR-POS-BGI15-BACARC,n/a,,n/a,n/a,,ENVO:00000150,,,USA,n/a,n/a,,,,,,,,,"coral skeleton, coral tissue, coral mucus",,,,,,,,,,,,,,,,,,,,y,DNeasy PowerSoil Kit,V4,515F/806R,,,,MiSeq,,Genewiz (NJ),
\ No newline at end of file
+25,,,,Salerno,FDEP SCTLD,JSEP_SCTLD_Endo,PCR-POS-BGI15-BACARC,n/a,,n/a,n/a,,ENVO:00000150,,,USA,n/a,n/a,,,,,,,,,"coral skeleton, coral tissue, coral mucus",,,,,,,,,,,,,,,,,,,,y,DNeasy PowerSoil Kit,V4,515F/806R,,,,MiSeq,,Genewiz (NJ),

```

Comment:
A new line was created and it might be a mistake.

Reply:
Ok I will withdraw this pull request